### DISCUSSION TITLE
Co pilot not offering suggestions

### DISCUSSION THREAD

**Discussion by ghost** [community/community#32839] [Copilot Conversations]
Hi, 

I have purchase GitHub Copilot and have installed the addon (v1.44.6735) in VS code. From what I can see it has been enabled (I see the copilot icon in the bottom right and am not being prompted for creds, etc). 

When I attempt the follow the getting started guide https://docs.github.com/en/copilot/getting-started-with-github-copilot/getting-started-with-github-copilot-in-visual-studio-code I receive no suggestion.

**Comment by davecheney** [comment]
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.
